kitsune/src/service/account.rs

```diff
@@ -73,12 +76,40 @@ pub struct GetPosts {
     #[builder(default = API_DEFAULT_LIMIT)]
     limit: usize,
 
-    /// Smallest ID
+    /// Smallest ID, return results starting from this ID
```

Comment:
Thanks for adding these docs here. I really had a hard time remembering the difference between `min_id` and `since_id` just now.

I really wish the Mastodon docs were better..

Reply:
I agree, I had to dig through github discussions to find that info lmao

Reply:
I think the first time I learned the distinction was looking at the docs of Mastodon.py heh